plugins/producer/main.go

```diff
@@ -82,6 +81,13 @@ func strPtr(x string) *string {
 	return &x
 }
 
+func getChainID() int64 {
```

Comment:
I'm kinda wondering if we shouldn't make this an xplugeth utility function. It seems like it will be a common case, we could get the off the Plugin Loader's singleton store, and just have like an `xplugeth.GetChainID()` function rather than have plugins have to replicate it. Thoughts?